src/Compilers/Core/Portable/Operations/PlaceholderKind.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    internal enum PlaceholderKind
+    {
+        Unspecified = 0,
+        SwitchOperationExpression = 1,
+        ForToLoopBinaryOperatorLeftOperand = 2,
+        ForToLoopBinaryOperatorRightOperand = 3,
+        AggregationGroup = 4,
+    }
```

Comment:
What is this for?

Reply:
It's used by IPlaceholderOperation. Previously this was defined in the same file as the operation itself, so I had to split it out.

Reply:
not the responsibility of the generator to create this?